phoenix-core/src/test/resources/hadoop-metrics2.properties

```diff
@@ -32,10 +32,9 @@
 #    [prefix].[source|sink].[instance].[options]
 # See javadoc of package-info.java for org.apache.hadoop.metrics2 for detail
 
-
-# Don't attempt to start jmx mbeans for all sources.
-#  For right now, all metrics are exported to a phoenix table
-*.source.start_mbeans=false
+phoenix.source.start_mbeans=true
+phoenix.sink.sink0.class=org.apache.phoenix.monitoring.GlobalPhoenixMetricsTestSink
```

Comment:
I think it usually starts with sink 1 .  (?)  at least ni HBase.

Reply:
It is just an identifier and can be anything.